deployment/deployment-config.ts

```diff
@@ -0,0 +1,96 @@
+import { isLocalStack } from "./local/util";
+import { getSecret } from "./utils/secrets-manager";
+
+export interface DeploymentConfigProperties {
+  project: string;
+  stage: string;
+  isDev: boolean;
+  vpcName: string;
+  oktaMetadataUrl: string;
+  bootstrapUsersPassword?: string;
+  launchDarklyClient: string;
+  redirectSignout: string;
+  cloudfrontCertificateArn?: string;
+  cloudfrontDomainName?: string;
+  secureCloudfrontDomainName?: string;
+  userPoolDomainPrefix?: string;
+  vpnIpSetArn?: string;
+  vpnIpv6SetArn?: string;
+  brokerString: string;
+  kafkaAuthorizedSubnetIds: string;
+}
+
+export const determineDeploymentConfig = async (stage: string) => {
+  const project = process.env.PROJECT!;
+  const isDev =
+    isLocalStack ||
+    !["master", "main", "val", "production", "jon-cdk"].includes(stage); // TODO: remove jon-cdk after main is deployed
```

Comment:
```suggestion
    !["main", "val", "production", "jon-cdk"].includes(stage); // TODO: remove jon-cdk after main is deployed
```
I could see an argument toward standardizing too so feel free to ignore.